docs/user/attestation-provider.md

```diff
@@ -0,0 +1,113 @@
+# Becoming an Attestation Provider
```

Comment:
APs are part of the infrastructure so this goes into `docs/infra/`.

The User section is mostly for user-facing UI stuff.

Reply:
Fixed in https://github.com/flare-foundation/docs/pull/14/commits/8893f83175ba5ffbc129f23ec752235b86f44841